### PR TITLE
CB-5878 add title for inline editor button

### PR DESCRIPTION
**Pull request by SychevAndrey**

### CODE REVIEW

File: webapp/packages/core-localization/src/locales/en.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export default [
   ['ui_custom', 'Custom'],
   ['ui_configure', 'Configure'],
   ['ui_apply', 'Apply'],
+  ['ui_reject', 'Reject'],
```
Comment (devnaumov) [inline webapp/packages/core-localization/src/locales/en.ts:44]: We probably could use just ui_cancel?

**Review by devnaumov — CHANGES_REQUESTED** [review]

File: webapp/packages/core-localization/src/locales/en.ts (hunk 1 above)
Comment (SychevAndrey) [reply webapp/packages/core-localization/src/locales/en.ts:44]: We discussed it with @Wroud and he was for adding Reject option. Anyway there is no cancel token in locales, there is only `processing_cancel`, so we need to add either Reject or Cancel

Comment (devnaumov) [reply webapp/packages/core-localization/src/locales/en.ts:44]: Then we need to change ru locale to "Отклонить"

**Review by devnaumov — APPROVED** [review]

File: webapp/packages/core-localization/src/locales/en.ts (hunk 1 above)
Comment (SychevAndrey) [reply webapp/packages/core-localization/src/locales/en.ts:44]: Personally I like more cancel option with "Отменить" translation. Can we discuss together and decide?

**Review by sergeyteleshev — APPROVED** [review]

**Pull request merged**